aes/ta/aes_ta.c

```diff
@@ -260,13 +260,13 @@ static TEE_Result set_aes_key(void *session, uint32_t param_types,
 	 * - reset transient object and load attribute data
 	 *   TEE_ResetTransientObject()
 	 *   TEE_PopulateTransientObject()
-	 * - load the key (transient object) into the cihering operation
+	 * - load the key (transient object) into the ciphering operation
 	 *   TEE_SetOperationKey()
 	 *
 	 * TEE_SetOperationKey() requires operation to be in "initial state".
 	 * We can use TEE_ResetOperation() to reset the operation but this
 	 * api cannot be used on operation with key(s) not yet set. Hence,
```

Comment:
@x8-999-github, as you are aournd here, could you also replace `api` with `API` here? thanks.